src/clj_cbor/codec.clj

```diff
@@ -922,7 +922,8 @@
         (error/*handler*
           ::unsupported-type
           (str "No known encoding for object: " (pr-str x))
-          {:value x})))
+          {:value x 
+           :type (type x)})))
```

Comment:
Two thoughts here - first, `class` is generally more appropriate than `type`, since the latter will return you `(:type (meta x))` which is (obviously) user-modifiable. Given that the default dispatch function is `class` this would also match the behavior better.

Second thought builds on the first one, which is that this case occurs when `write-handled` falls through because there's no handler for the dispatched type. It may be more useful to make this data include the `dispatch` function's return value, since that's what's actually used for the handler behavior.